pytorch_lightning/callbacks/progress/rich_progress.py

```diff
@@ -289,14 +285,18 @@ def _init_progress(self, trainer):
             self.progress = CustomProgress(
                 *self.configure_columns(trainer),
                 self._metric_component,
-                refresh_per_second=self.refresh_rate_per_second,
+                auto_refresh=False,
                 disable=self.is_disabled,
                 console=self._console,
             )
             self.progress.start()
             # progress has started
             self._progress_stopped = False
 
+    def refresh(self):
```

Comment:
```suggestion
    def refresh(self) -> None:
```

could you add mypy/typehint coverage for this class in a separate PR?

Reply:
Yup!